lib/parsers/convert_to_rspec.rb

```diff
@@ -0,0 +1,229 @@
+# Taken from https://gist.github.com/seven1m/e375bcdf2864da0022f1
+
+require "parser/current"
+require "unparser"
+require "fileutils"
+
+class ConvertToRspec
+  attr_reader :minitest_folder, :rspec_folder
+
+  def initialize(minitest_folder, rspec_folder)
+    @minitest_folder = minitest_folder
+    @rspec_folder = rspec_folder
+  end
+
+  def go!
+    Dir["#{minitest_folder}/*_test.rb"].each do |test|
+      puts test
+      body = File.read(test)
+      test.gsub!(minitest_folder.to_s, rspec_folder.to_s)
+      new_path = test.gsub!(/_test\.rb$/, "_spec.rb")
+      # custom_fixes_before!(body, new_path)
+      rails_helper!(body, new_path)
```

Comment:
As `rails_helper` as been merged with `spec_helper`, I don't think this is needed.